### PR TITLE
Bootstrap upgrade bugfixes 2

### PR DESCRIPTION
**Pull request by ashwiniHerle**
Minor styling fixes related to bootstrap upgrade.
Made analyses look similar throughout samples, reactions and researchplans, wellplates and cellLines
Fix for issue where contextActions buttons vertically expanded in smaller screens
Show notification message numbers on NoticeButton

### CODE REVIEW

File: app/packs/src/apps/mydb/elements/details/researchPlans/analysesTab/ResearchPlanDetailsContainers.js [hunk 1]
```diff
@@ -269,28 +269,37 @@ export default class ResearchPlanDetailsContainers extends Component {
...
+              onSelect={this.handleAccordionOpen}
+              activeKey={activeContainer}
+            >
               {analysesContainer[0].children.map((container, key) => {
                 if (container.is_deleted) {
                   return (
                     <Accordion.Item
                       eventKey={key}
                       key={`research_plan_container_deleted_${container.id}`}
+                      className="border rounded overflow-hidden"
```
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/researchPlans/analysesTab/ResearchPlanDetailsContainers.js:283]: The border is too big. Maybe border rounded is not necessary here

File: app/packs/src/apps/mydb/elements/details/researchPlans/analysesTab/ResearchPlanDetailsContainers.js
Comment (beque) [inline app/packs/src/apps/mydb/elements/details/researchPlans/analysesTab/ResearchPlanDetailsContainers.js:1]: Buttons in Analyses Header have no gap

**Review by beque — COMMENTED** [review]
Just some small fixes.

I have seen that the Analyses Header of CellLine and Wellplate still look different

File: app/packs/src/components/sampleTaskInbox/SampleTaskNavigationElement.js [hunk 2]
```diff
@@ -3,16 +3,16 @@ import { observer } from 'mobx-react';
 import { Button, Badge } from 'react-bootstrap';
 import { StoreContext } from 'src/stores/mobx/RootStore';
 
-const SampleTaskNavigationElement = ({}) => {
+function SampleTaskNavigationElement() {
```
Comment (beque) [inline app/packs/src/components/sampleTaskInbox/SampleTaskNavigationElement.js:6]: Why have you changed this?
`const SampleTaskNavigationElement = ({}) => ` is correct

**Review by beque — COMMENTED** [review]
Just 2 analyses components with double border and one question.
When you fixed this you can merge this PR.

The navigation buttons looking good on laptop and big screen

File: app/packs/src/components/sampleTaskInbox/SampleTaskNavigationElement.js (hunk 2 above)
Comment (ashwiniHerle) [reply app/packs/src/components/sampleTaskInbox/SampleTaskNavigationElement.js:6]: There were linting errors. I'll undo it

**Pull request merged**